src/main/features/offer/views/declaration.njk

```diff
@@ -74,6 +72,22 @@
 
       <h2 class="heading-medium">{{ t('Sign the agreement') }}</h2>
 
+      <p>
+        {% if user.id == claim.defendantId %}
+          {{
+            t('Make sure this contract includes everything you have agreed with {{ claimantName }} before signing.', {
```

Comment:
Please use translation and wording only once, make name placeholder more generic like `name` and based on condition pass either claimant name or defendant name.